### PR TITLE
#1615 Fix XML initialization for some unit tests

### PR DESCRIPTION
**Pull request by valleefred**

### CODE REVIEW

File: dynawo/sources/API/CRT/test/TestXmlImporter.cpp [hunk 1]
```diff
@@ -1,14 +1,14 @@
 //
-// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline dynawo/sources/API/CRT/test/TestXmlImporter.cpp:2]: year of all copyright should not be changed when you edit a file.

File: dynawo/sources/API/CRV/test/TestXmlImporter.cpp [hunk 2]
```diff
@@ -1,14 +1,14 @@
 //
-// Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline dynawo/sources/API/CRV/test/TestXmlImporter.cpp:2]: same

File: dynawo/sources/API/DYD/test/Test.cpp [hunk 3]
```diff
@@ -1,14 +1,14 @@
 //
-// Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline dynawo/sources/API/DYD/test/Test.cpp:2]: same

File: dynawo/sources/Test/initXml.cpp [hunk 4]
```diff
@@ -1,14 +1,14 @@
 //
-// Copyright (c) 2021, RTE (http://www.rte-france.com)
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline dynawo/sources/Test/initXml.cpp:2]: When a new file is created the year of the copyright should be the year it is created so don't add 2015, and this file was created in 2021.

**Review by gautierbureau — CHANGES_REQUESTED** [review]

**Comment by gautierbureau** [comment]
I would prefer if you could squash the two commits before merging it. thanks

**Comment by valleefred** [comment]
> I would prefer if you could squash the two commits before merging it. thanks

Thanks, I've just learned how to squash ! ;-)

**Review by gautierbureau — APPROVED** [review]

File: dynawo/sources/API/CRT/test/TestXmlImporter.cpp [hunk 5]
```diff
@@ -7,8 +7,8 @@
 // file, you can obtain one at http://mozilla.org/MPL/2.0/.
 // SPDX-License-Identifier: MPL-2.0
 //
-// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
-// simulation tool for power systems.
+// This file is part of Dynawo, an hybrid C++/Modelica open source suite
```
Comment (rosiereflo) [inline dynawo/sources/API/CRT/test/TestXmlImporter.cpp:10]: Please do not modify the copyrights of existing files

**Review by rosiereflo — CHANGES_REQUESTED** [review]

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**